wolfssl/wolfio.h

```diff
@@ -39,7 +39,7 @@
 
 #if !defined(WOLFSSL_USER_IO)
     /* Micrium uses NetSock I/O callbacks in wolfio.c */
-    #if !defined(USE_WOLFSSL_IO) && !defined(MICRIUM)
+    #if !defined(USE_WOLFSSL_IO) && !defined(MICRIUM) && !defined(CONTIKI)
```

Comment:
Is this supposed to be `WOLFSSL_CONTIKI`?

Reply:
fixed